src/lib/DataTable/DataTableRouted.tsx

```diff
@@ -35,6 +39,11 @@ export function DataTableRouted<T, TFilter, TRouteNames>({
   const [orderState, setOrderState] = useState<OrderOption>({ orderBy: columns[0].dataField, asc: true });
   const filterRefs = useRef<Filters>({});
 
+  if (handlers)
```

Comment:
Correct me if im wrong but this is executed on every render now. I think doing it once would be sufficient? A useEffect with an empty dependecy array should do the trick.

Reply:
You're right, did not think about that. Added `useEffect`

Reply:
Did you test it with the new code and when you are for example on page 2 or already have a filter set? 
I'm afraid that closures might take a snapshot of the variables and therefor misbehave and always pass the initial values to the `loadPage` function. But i'm not 100% sure.

Reply:
Unfortunately it did snapshot all the arguments, therefore I removed `useEffect`.

Reply:
I would highly suggest to write a jest test for this. (Checking if the supplied arguments to the load page are correct)

Still unsure if there is not a better option than doing it on every render. Seems a bit overkill but maybe i'm wrong. 
IIRC refs would work. But than you would have to keep a state and a ref which also seems a bit stupid...

Reply:
Updated snapshots and added reloading tests